components/bl-pagination-component/component.json

```diff
@@ -0,0 +1,149 @@
+{
+  "id": "c_1505076c978bd1a43a67369c0f255644",
+  "name": "Pagination",
+  "description": "Pagination is a component for switching between pages",
+  "showInToolbox": true,
+  "faIcon": "chevron-circle-right",
+  "mainJS": "dist/index.js",
+  "type": "custom",
+  "category": "Custom Components",
+  "properties": [
+    {
+      "type": "number",
+      "name": "countPages",
+      "label": "Count Pages",
+      "handlerId": "CountPagesLogic",
+      "handlerLabel": "Count Pages Logic",
+      "handlerDescription": "This is a handler for the logic to determine the count of pages.",
+      "defaultValue": "10"
+    },
+    {
+      "type": "number",
+      "name": "siblingCount",
+      "label": "Sibling Count",
+      "defaultValue": "1",
+      "handlerId": "siblingCountLogic",
+      "handlerLabel": "Sibling Count Logic",
+      "handlerDescription": "This is a handler for the logic to determine how many digits to display either side of current page."
+    },
+    {
+      "type": "select",
+      "name": "variant",
+      "label": "Variant",
+      "options": [
+        {
+          "value": "normal",
+          "label": "Normal"
+        },
+        {
+          "value": "outline",
+          "label": "Outline"
+        },
+        {
+          "value": "rounded",
+          "label": "Rounded"
+        },
+        {
+          "value": "rounded-outline",
+          "label": "Rounded-Outline"
+        }
+      ],
+      "defaultValue": "normal"
+    },
+    {
+      "type": "select",
+      "name": "size",
+      "label": "Size",
+      "defaultValue": "",
+      "options": [
+        {
+          "value": "small",
+          "label": "Small"
+        },
+        {
+          "value": "",
```

Comment:
I'm not sure, but sometime we have a problem with empty values in designer settings. Could you check if changing this property in designer works good?